cirq-core/cirq/ops/common_gates_test.py

```diff
@@ -899,6 +899,8 @@ def test_cphase_unitary(angle_rads, expected_unitary):
     np.testing.assert_allclose(cirq.unitary(cirq.cphase(angle_rads)), expected_unitary)
 
 
+# TODO(#6663): fix this use case.
+@pytest.mark.xfail
```

Comment:
Do you know why this stopped working? If just the latter assertion is broken can we break it out into it's own test and mark it as failed / track a fix for it, WDYT?

Reply:
looks like either comparison between sympy numbers and floats changed or something changed that changes the functions called during comparison ... either way I opened https://github.com/quantumlib/Cirq/issues/6663 to fix it.